.azure-pipelines.yml

```diff
@@ -76,26 +69,18 @@ jobs:
       #       "msvc-v16.10.0" | build
       #     cacheHitVar: BUILD_CACHE_RESTORED
 
-      - task: UsePythonVersion@0
-        inputs:
-          versionSpec: '3.x' 
-          addToPath: true 
-          architecture: "$(TRIPLET)"
-
       - script: |
           git clone https://github.com/microsoft/vcpkg %VCPKG_DIR%
-          git -C %VCPKG_DIR% checkout %VCPKG_REF%
+          git checkout %VCPKG_REF%
```

Comment:
nooooo, ffs..  I built its master accidentally.. damn, will have to invalidate the cache once again.